### PR TITLE
feat(launcher): add default url to launched browser

### LINKED ISSUE #2402: Different initial pages in different modes
* Puppeteer version: v1.3.0-post
* Platform / OS version: Windows 7 x64
* Node.js version: 10.0.0 v8-canary

```js
(async () => {
  const browser = await require('puppeteer').launch();
  console.log((await browser.pages()).map(page => page.url()));
  await browser.close();
})();
```
```js
[ 'about:blank' ]
```
```js
(async () => {
  const browser = await require('puppeteer').launch({ headless: false });
  console.log((await browser.pages()).map(page => page.url()));
  await browser.close();
})();
```
```js
[ 'chrome-search://local-ntp/local-ntp.html' ]
```

Would it be more consistent, safe and performance-wise optimal to load `about:blank` initially in both modes?

### PR DESCRIPTION
**Pull request by yanivefraim**
fixes #2402

### CODE REVIEW

**Review by JoelEinbinder — APPROVED** [review]
Thanks!

File: test/puppeteer.spec.js [hunk 1]
```diff
@@ -264,6 +264,18 @@ module.exports.addTests = function({testRunner, expect, PROJECT_ROOT, defaultBro
         await page.close();
         await browser.close();
       });
+      it('should have default url when launching browser', async function() {
+        const browser = await puppeteer.launch();
```
Comment (JoelEinbinder) [inline test/puppeteer.spec.js]: Please use the default browsing arguments here. See other tests that launch chrome.

**Comment by yanivefraim** [comment]
@JoelEinbinder - thanks for approving. My tests are broken on Linux, I'm about to push a fix.

File: test/puppeteer.spec.js (hunk 1 above)
Comment (yanivefraim) [reply test/puppeteer.spec.js]: yes, I figured this out. pushed a fix. 10x!

**Review by JoelEinbinder — CHANGES_REQUESTED** [review]
Actually, this might burn people that were using the arguments to launch Chromium to a default page. Now they will get two tabs, and their tab will no longer be the first entry in browser.pages(). I think we  have to do something fancy and only append this argument if no other page was specified in the arguments.

**Comment by yanivefraim** [comment]
@JoelEinbinder - I understand. Good catch! 

So, I will have to look for an argument without `--` at start maybe (not sure this is a good indicator, I'll have to look at the docs), and in case it is present, I'll skip default `about:blank`? WDYT?

**Comment by yanivefraim** [comment]
After looking at [chromium-command-line-switches](https://peter.sh/experiments/chromium-command-line-switches/), I think that the solution I suggested might work... (although it is kind of ugly)

**Comment by JoelEinbinder** [comment]
Yes, anything starting with '-' should be a URL in the chrome command line args.

**Comment by yanivefraim** [comment]
@JoelEinbinder - added a fix for existing url in launch args. 10x!

**Review by JoelEinbinder — APPROVED** [review]

**Pull request merged**